README.md

```diff
@@ -230,3 +230,49 @@ $ yarn start notifications send-user-info-notification \
   --url='http://USER:PASSWORD@HOST:PORT' \
   usernames.csv emailContent.json
 ```
+
+## Users
+
+### Migrate user email to username
```

Comment:
That's sound a bit unclear, we are changing also the username, right?

What if we want to change the username but it's not an email?